src/main/java/amazonreviewsapp/springboot/model/User.java

```diff
@@ -0,0 +1,74 @@
+package amazonreviewsapp.springboot.model;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.Id;
+import javax.persistence.ManyToMany;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+import java.util.Set;
+
+@Entity
+@Table(name = "users")
+public class User {
+    @Id
+    private String userId;
+    private String profileName;
+    private String profilePassword;
+    @ManyToMany(fetch = FetchType.LAZY, cascade = CascadeType.ALL)
```

Comment:
if one user will be deleted it may delete all roles related to him. Other user may have those roles too so let's not  use CascadeType.ALL

Reply:
CascadeType changed to Persist